src/features/nft/nftModals/bidBuyModal.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import { InputNumberFormat } from '../../../common/components/Inputs'
+
+export type TBidBuyModal = {
+  username?: string
+  isOpen: boolean
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const BidBuyModal = ({
+  username,
+  isOpen,
+  handleClose,
+}: TBidBuyModal): JSX.Element => {
+  return (
+    <Modal isOpen={isOpen} handleClose={handleClose} size='lg'>
+      <h2 className='mb-6'>Buy now</h2>
+      <p className='mb-6'>
+        You are buy{' '}
+        <span className='font-medium text-blue-3f'>
+          Super Nfty Floating Head Professional
+        </span>{' '}
+        from <span className='font-medium text-blue-3f'>{username}</span>
+      </p>
+      <div className='mb-6'>
+        <InputNumberFormat
+          label='Enter quantity'
+          hint='3 available'
+          type='text'
+          placeholder='Number'
+          append={<span className='text-gray-a0'>PSL</span>}
```

Comment:
Can we get the name(PSL) from `currencyName` because we have a testnet and the `PSL` will change to `PLS`. Please also update for below sections